### PR TITLE
sbsigntools + efitools for UEFI secure boot

### PR DESCRIPTION
**Pull request by congdanhqx**
- Compiled on x86_64 for:
    - x86_64
    - aarch64
    - aarch64-musl

- Run with secure boot enabled on:
    - x86_64 on ThinkPad T480

### CODE REVIEW

File: srcpkgs/efitools/template [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+else
+	hostmakedepends+=" help2man"
+fi
+
+post_patch() {
+	[ ! "$CROSS_BUILD" ] && return
+
+	case "${XBPS_TARGET_MACHINE}" in
+		arm*) _ARCH=arm ;;
+		i?86*) _ARCH=ia32 ;;
```
Comment (Johnnynator) [inline srcpkgs/efitools/template]: we only have `i686*` so the `?` is not needed..

File: srcpkgs/sbsigntool/template [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Template file for 'sbsigntool'
+pkgname=sbsigntool
+version=0.9.2
+revision=1
+#only_for_archs="i686 x86_64"
```
Comment (Johnnynator) [inline srcpkgs/sbsigntool/template]: Remaining useless comment, remove it.

File: srcpkgs/efitools/template [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+makedepends="gnu-efi-libs libressl-devel"
+short_desc="Tools to manipulate EFI secure boot platforms"
+maintainer="Doan Tran Cong Danh <congdanhqx@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git"
+distfiles="https://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git/snapshot/${pkgname}-${version}.tar.gz"
+checksum=0f315b36e7d1ba74bfc97ab9f304f0a3072c47578bbe5e42594acae381f9acfe
+patch_args="-Nup1"
+
+subpackages="efitools-efi"
```
Comment (Johnnynator) [inline srcpkgs/efitools/template]: you don't need to set subpackages manually if the subpackage is build always. This is only needed for packages that not always build all subpackages.

File: srcpkgs/sbsigntool/template [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+if [ ! "$CROSS_BUILD" ]; then
+	hostmakedepends+=" help2man"
+fi
+
+pre_patch() {
+	[ ! "$CROSS_BUILD" ] && return
+
+	case "${XBPS_TARGET_MACHINE}" in
+		arm*) _ARCH=arm ;;
+		i?86*) _ARCH=ia32 ;;
```
Comment (maxice8) [inline srcpkgs/sbsigntool/template]: i686*

**Pull request closed without merging**